FWCore/MessageLogger/src/ELseverityLevel.cc

```diff
@@ -188,6 +150,7 @@ namespace edm {
       ret[ELsev_zeroSeverity] = "ELzeroSeverity   ";
       ret[ELsev_success] = "ELdebug          ";  // 4/21/09
```

Comment:
Do you want to remove the comment? (I'm fine with either way)

Reply:
done